src/components/Modals/NewCourse.vue

```diff
@@ -1,24 +1,102 @@
 <template>
   <div class="newCourse">
     <div v-if="!isOnboard" class="newCourse-text">{{ text }}</div>
+    <!-- TODO: for some reason this breaks the dropdown <div v-if="selected" class="newCourse-name newCourse-requirements-container">{{ selectedCourse }}</div> -->
     <div class="autocomplete">
       <input class="newCourse-dropdown" :id="'dropdown-' + semesterID" :ref="'dropdown-' + semesterID" :placeholder="placeholder" @keyup.enter="addCourse" @keyup.esc="closeCourseModal" />
     </div>
+    <div v-if="!isOnboard && !selected"> <!-- if a course is not selected -->
+      <div class="newCourse-title">Add this class to the following semester</div>
+      <div class="newCourse-semester-edit">
+        <newSemester :type="season" :year="year"></newSemester>
```

Comment:
super nitpicky but when a season is chosen the emoji disappears 🥺 can we still have it appear? 
<img width="365" alt="Screen Shot 2020-11-06 at 11 33 57 AM" src="https://user-images.githubusercontent.com/14082828/98391217-5a403180-2024-11eb-9ac5-4fb4488010e7.png">

Reply:
I will address this in a future PR since this is factored out for now.